Source/Fuse.Controls.ScrollView/Scroller.uno

```diff
@@ -198,9 +200,26 @@ namespace Fuse.Gestures
 		{
 			get { return _scrollable == null ? GesturePriority.Lower : _scrollable.GesturePriority; }
 		}
+
+		bool _pressed;
+		void StartInvalidateVisual()
+		{
+			if (!_pressed)
+				UpdateManager.AddAction(_scrollable.InvalidateVisual);
```

Comment:
@mortoray: We don't, not one that propagates up the tree like we would need here. I think we should make a mechanism like that instead. But maybe something like this could be an OK stop-gap solution.